client/src/components/Layout/AppNavbar.vue

```diff
@@ -8,27 +8,100 @@
           src="@/assets/images/logo-techtonic.png"
         />
       </router-link>
-      <span>
-        <router-link class="navbar__link" to="/">Home</router-link>
-        <router-link class="navbar__link" to="/reports">Reports</router-link>
-        <router-link class="navbar__link" to="/about">About</router-link>
-        <router-link class="navbar__link" to="/company-values"
-          >Company Values</router-link
+      <span class="navbar__links">
+          <router-link class="navbar__link navbar__link--heading" to="/">Home</router-link>
```

Comment:
`navbar__link--heading` doesn't appear to be used...Is this needed for anything?